cms/djangoapps/contentstore/signals.py

```diff
@@ -39,13 +39,17 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
 
     # Finally call into the course search subsystem
     # to kick off an indexing action
-
     if CoursewareSearchIndexer.indexing_is_enabled():
         # import here, because signal is registered at startup, but items in tasks are not yet able to be loaded
         from .tasks import update_search_index
-
         update_search_index.delay(unicode(course_key), datetime.now(UTC).isoformat())
 
+    # If the Git auto-export is enabled, push the course changes to Git
```

Comment:
`ENABLE_EXPORT_GIT` allows the user to export the course contents to git by button-push, and `ENABLE_GIT_AUTO_EXPORT` automatically exports the course contents to git every time the course is published